package.json

```diff
@@ -13,17 +13,17 @@
     "d3": "^5.15.1",
     "flat": "^5.0.0",
     "hooks": "^0.3.2",
-    "husky": "^4.2.3",
+    "husky": "^4.2.5",
     "jss-rtl": "^0.3.0",
-    "lint-staged": "^10.1.2",
+    "lint-staged": "^10.1.3",
     "node-sass": "^4.13.1",
     "papaparse": "^5.2.0",
-    "prettier": "^2.0.2",
+    "prettier": "^2.0.4",
```

Comment:
Was there a reason for updating these packages? Did the `react-intl` require these packages to be updated?